CONTRIBUTING.md

```diff
@@ -169,24 +198,35 @@ subcommands by running:
 
     gno --help
 
+#### Adding new tests
+
+Most packages will follow the convention established with Go: each package
+contains within its file many files suffixed with `_test.go` which test its
+functionality. As a general rule, you should follow this convention, and in
+every PR you make you should ensure all the code you added is appropriately
+covered by tests ([Codecov](https://about.codecov.io/) will loudly complain in
+your PR's comments if you don't).
+
+Additionally, we have a few testing systems that stray from this general rule;
+at the time of writing, these are for integration tests and language tests. You
+can find more documentation about them [on this guide](gno/docs/testing-guide.md).
+
 ### Repository Structure
 
 The repository structure can seem tricky at first, but it’s simple if you consider the philosophy that the gno project
-employs (check out [PHILOSOPHY.md](https://github.com/gnolang/gno/blob/master/PHILOSOPHY.md)).
+employs (check out [PHILOSOPHY.md](./PHILOSOPHY.md)).
```

Comment:
I think it is currently ../. But it makes sense to move to docs/.

btw, @zivkovicmilos, please make sure to follow recent changes in the docs/ folder for your PR.

Reply:
Well, CONTRIBUTING is top level, so is PHILOSOPHY. So `./` makes sense here. Do we want to move PHILOSOPHY & PLAN to `docs/`?

Reply:
Not sure, we can keep here for now.